lib/ansible/modules/network/meraki/meraki_switchport.py

```diff
@@ -33,21 +33,19 @@
     allowed_vlans:
         description:
         - List of VLAN numbers to be allowed on switchport.
-        default: all
+        - Relevant only for trunk ports.
+        - Defaults to C(all) if not specified.
     enabled:
         description:
         - Whether a switchport should be enabled or disabled.
         type: bool
-        default: yes
     isolation_enabled:
         description:
         - Isolation status of switchport.
-        default: no
```

Comment:
Well, my remark was not just for this one parameter, but for all parameters.
And also for all Meraki modules.